Classes/Domain/Model/SecondFactor.php

```diff
@@ -38,6 +38,12 @@ class SecondFactor
      */
     protected string $secret;
 
+    /**
+     * @var \DateTime
+     * @ORM\Column(type="datetime", nullable=false)
```

Comment:
isn't the `creationDate` nullable as it does not exist currently? Not sure what the database does when adding a new, not-nullable column to data entries not existing yet